packages/patrol_cli/lib/src/test_finder.dart

```diff
@@ -59,10 +64,12 @@ class TestFinder {
   }
 
   /// Recursively searches the `integration_test` directory and returns files
-  /// ending with `_test.dart` as absolute paths.
+  /// ending with defined [testFileSuffix]. If [testFileSuffix] is not defined,
+  /// the default suffix `_test.dart` is used.
   List<String> findAllTests({
     Directory? directory,
     Set<String> excludes = const {},
+    String? testFileSuffix,
```

Comment:
Can you make `testFileSuffix` parameter non-nullable and make it default to `_kDefaultTestFileSuffix`?